nullaway/src/main/java/com/uber/nullaway/handlers/InferredJARModelsHandler.java

```diff
@@ -91,13 +94,40 @@ public InferredJARModelsHandler() {
     }
   }
 
+  /*
+   * Scan Java Classpath for JarInfer model jars and map their names to locations
+   */
+  private void processClassPath(String[] args) {
+    Set<String> paths = new LinkedHashSet<>();
+    for (int i = 0; i < args.length; ++i)
+      if (args[i].equals("-classpath") || args[i].equals("-processorpath"))
```

Comment:
Maybe we should also add `-cp` (an alias for `-classpath`) and read the `CLASSPATH` environment variable if set? I think we don't use those internally, but they make sense for open source.